alert-common/src/main/java/com/synopsys/integration/alert/common/message/model/LinkableItem.java

```diff
@@ -91,7 +102,12 @@ public void setSummarizable(final boolean summarizable) {
 
     @Override
     public int compareTo(final LinkableItem otherItem) {
-        return CompareToBuilder.reflectionCompare(this, otherItem);
+        if (collapsible && !otherItem.collapsible) {
+            return 1;
```

Comment:
Do we want to use the constant here or is 1 enough?

Reply:
1 should be the expected return type